network/tailscale/README.md

```diff
@@ -21,11 +21,31 @@ environment:
   - TS_AUTHKEY=<your auth key>
 ```
 
+You need to append the above YAML to your main configuration.
+
+```yaml
+machine:
+  type: controlplane
+...(rest of your node config)
+---
+apiVersion: v1alpha1
+kind: ExtensionServiceConfig
+name: tailscale
+...
+```
+
+Then apply the full configuration to your node with
 ```bash
-> talosctl apply -n node myconfig.yaml
-> talosctl upgrade -n node
+talosctl apply -n mynode -f mynodeconfig.yaml
```

Comment:
i think we can just do `talosctl patch mc @patch.yaml`

Reply:
yes, the easiest way to add a multi-doc config is to patch your config; if the doc doesn't exist, it will be appended; if it already exists, it will be updated

Reply:
I updated the PR with your feedback. Thanks.